### PR TITLE
t1m0thyj.WinDynamicDesktop version 4.6.0.0

### PR DESCRIPTION
**Pull request by OfficialEsco**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17433)

### CODE REVIEW

File: manifests/t/t1m0thyj/WinDynamicDesktop/2.6.0.0/t1m0thyj.WinDynamicDesktop.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate
+
+PackageIdentifier: t1m0thyj.WinDynamicDesktop
+PackageVersion: 2.6.0.0
```
Comment (ItzLevvie) [inline manifests/t/t1m0thyj/WinDynamicDesktop/2.6.0.0/t1m0thyj.WinDynamicDesktop.installer.yaml:4]:
```suggestion
PackageVersion: 4.6.0.0
```

File: manifests/t/t1m0thyj/WinDynamicDesktop/2.6.0.0/t1m0thyj.WinDynamicDesktop.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate
+
+PackageIdentifier: t1m0thyj.WinDynamicDesktop
+PackageVersion: 2.6.0.0
```
Comment (ItzLevvie) [inline manifests/t/t1m0thyj/WinDynamicDesktop/2.6.0.0/t1m0thyj.WinDynamicDesktop.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 4.6.0.0
```

File: manifests/t/t1m0thyj/WinDynamicDesktop/2.6.0.0/t1m0thyj.WinDynamicDesktop.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: t1m0thyj.WinDynamicDesktop
+PackageVersion: 2.6.0.0
```
Comment (ItzLevvie) [inline manifests/t/t1m0thyj/WinDynamicDesktop/2.6.0.0/t1m0thyj.WinDynamicDesktop.yaml:4]:
```suggestion
PackageVersion: 4.6.0.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see my review.

File: manifests/t/t1m0thyj/WinDynamicDesktop/2.6.0.0/t1m0thyj.WinDynamicDesktop.installer.yaml (hunk 1 above)
Comment (OfficialEsco) [reply manifests/t/t1m0thyj/WinDynamicDesktop/2.6.0.0/t1m0thyj.WinDynamicDesktop.installer.yaml]: Just a very big typo 😬

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**